examples/evaluation_and_profiling/swe_bench/src/nat_swe_bench/configs/config_iterative.yml

```
@@ -0,0 +1,82 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+llms:
+ nim_llm:
+   _type: nim
+   model_name: mistralai/mistral-nemotron
+   temperature: 0.6
+   max_tokens: 4096    
+
+llms:
```

YAMLlint (1.37.1)

[error] duplication of key "llms" in mapping

(key-duplicates)